src/test/groovy/com/github/maleksandrowicz93/cqrsdemo/student/StudentFacadeSpec.groovy

```diff
@@ -0,0 +1,266 @@
+package com.github.maleksandrowicz93.cqrsdemo.student
+
+
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.StudentDto
+import com.github.maleksandrowicz93.cqrsdemo.student.exception.InvalidCredentialsException
+import com.github.maleksandrowicz93.cqrsdemo.student.exception.StudentAlreadyExistsException
+import com.github.maleksandrowicz93.cqrsdemo.student.exception.StudentNotFoundException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ContextConfiguration
+import spock.lang.Specification
+
+@SpringBootTest
+@ContextConfiguration
+class StudentFacadeSpec extends Specification {
```

Comment:
nie duplikują te testy przypadkiem tego, co jest w testach kontrolera?

Reply:
trochę duplikują, ale to, że fasada coś dobrze zwraca, nie znaczy jeszcze, że API zwraca prawidłowe wyniki - chcę mieć tutaj pewność, że dostaję z API dokładnie to, czego się spodziewam

Reply:
Jak skończysz dev i poprawki, to wrzucę kilka dodatkowych wymagań, żebyś sprawdzić kodzik - IMO duplikacja w testach i testowanie wszystkiego na każdym poziomie będzie problematyczne i czasochłonne w przyszłości ;)